fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/mixin/content/registry/MixinAxeItem.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.content.registry;
+
+import java.util.Map;
+
+import com.google.common.collect.Maps;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Mutable;
+import org.spongepowered.asm.mixin.Shadow;
+
+import net.minecraft.block.Block;
+import net.minecraft.item.AxeItem;
+
+@Mixin(AxeItem.class)
+public class MixinAxeItem {
+	@Shadow
+	@Final
+	@Mutable
+	protected static Map<Block, Block> STRIPPED_BLOCKS;
+
+	static {
+		STRIPPED_BLOCKS = Maps.newHashMap(STRIPPED_BLOCKS); // Makes the map mutable
```

Comment:
I don't know what the policy is on dependencies, but you could just as well write `new HashMap<>(STRIPPED_BLOCKS)` and then you wouldn't require Guava.

Reply:
This method is like created because before java 7 you cannot use `new HashMap<>` diamond, and now this is no longer a problem and the constructor is preferred.

Reply:
`new HashMap<>` is better anyway, but I think Guava is fine since it comes with Minecraft.